src/applications/planning-guidance/8832/28-8832-schema.json

```diff
@@ -0,0 +1,216 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "28-8832 Personal career planning and guidance (28-8832)",
+  "type": "object",
+  "additionalProperties": false,
+  "definitions": {
+    "address": {
+      "type": "object",
+      "oneOf": [
+        {
+          "properties": {
+            "country": {
```

Comment:
Where does this schema come from? Will address always be limited to the USA, Canada, and Mexico?

Reply:
All of the code in here is boilerplate code